src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -237,6 +242,6 @@ public void parseSortOptions_invalidOption_throwsParseException() {
     @Test
     public void parseSortOptions_validOption_returnsSortOptions() throws Exception {
         SortOptions expectedOption = SortOptions.PRIORITY_ASCENDING;
-        assertEquals(expectedOption, ParserUtil.parseSortOptions("priority", "a"));
+        assertEquals(expectedOption, parseSortOptions("priority", "a"));
```

Comment:
I think can just leave it in the old form as ParserUtil is static but looks fine